app/views/admin/works/index.html.erb

```diff
@@ -54,12 +54,20 @@
     </div>
 
     <div class="admin-filter">
-      <label for="q_parent_id_null">Child Works In Results</label>
```

Comment:
Thanks for the label change. Don't know why I got so confused last week but this is easier to understand.

Reply:
Ha, to me the new label is _harder_ to understand, I changed it just to save space to add another control in, worried that it would be harder to understand!